compiler/compiler.go

```diff
@@ -1100,7 +1106,7 @@ func (c *Compiler) parseInstr(frame *Frame, instr ssa.Instruction) {
 			params = append(params, context) // context parameter
 			switch c.selectScheduler() {
 			case "coroutines":
-				params = append(params, llvm.ConstPointerNull(c.i8ptrType)) // parent coroutine handle
+				// nothing special here
```

Comment:
"nothing special here" is not very informative. Can you add a comment instead that explains why nothing should be added as a parameter?